style.css

```diff
@@ -549,6 +549,25 @@ textarea {
     font-family: "Poppins", sans-serif;
   }
 
+  /* navbar section */
+  .nav-item {
+    display: block;
+  }
+
+  .nav-item a {
+    color: #344563;
+    font-weight: 500;
+  }
+
+  .nav-menu {
+    padding-right: 70px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  /* header section */
   .hero-section {
     background-image: url(assets/images/desktop-bg-img.png);
     background-repeat: no-repeat;
```

Comment:
- [ ] Please add more padding to the top of your `hero-section` it's too compact, kindly use the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848) to decide how much to add. 😁

Reply:
@mirouhml Thank you for this review. I will do that now.